.github/workflows/performance.yml

```diff
@@ -0,0 +1,56 @@
+name: Performance Test
+
+on:
+  pull_request:
+    branches: [master, 'release/**']
+    paths-ignore:
+      - 'docs/**'
+      - '**/*.md'
+
+jobs:
+  performance:
+    if: github.event_name == 'pull_request' || github.event.label.name == 'performance'
+    runs-on: ubuntu-18.04
+    timeout-minutes: 45
+
+    steps:
+      - name: Check out code
+        uses: actions/checkout@v2.4.0
+        with:
+          submodules: recursive
+
+      - name: Cache deps
+        uses: actions/cache@v2.1.7
+        env:
+          cache-name: cache-deps
+        with:
+          path: deps
+          key: ${{ runner.os }}-${{ env.cache-name }}-${{ matrix.os_name }}-${{ hashFiles('rockspec/apisix-master-0.rockspec') }}
+
+      - name: Install Dependencies
+        run: sudo ./ci/performance_test.sh install_dependencies
+
+      - name: Install wrk2
+        run: sudo ./ci/performance_test.sh install_wrk2
+
+      - name: Install SystemTap Tools
+        run: sudo ./ci/performance_test.sh install_stap_tools
+
+      - name: Perf Test
+        run: ./ci/performance_test.sh run_performance_test
```

Comment:
Is it possible to use `sudo` here and then remove `sudo` from the script?

Reply:
Tried it before and it didn't fit

Reply:
OK